script.js

```diff
@@ -0,0 +1,34 @@
+const header = document.querySelector('header');
+const title = document.getElementById('logo');
+const button = document.querySelector('.menu-bars');
+
+// const main = document.querySelector("main");
+
+const navbar = document.querySelector('nav');
+const cross = document.createElement('button');
+cross.innerHTML = '<i class="fas fa-times fa-2x"></i>';
+
+const link = document.querySelector('.nav-link');
+const link2 = document.querySelector('.link2');
+const link3 = document.querySelector('.link3');
```

Comment:
- [ ] It would be better if you name these variables with meaningful names because it helps you to work easily. See this [article](https://zelig880.com/coding-standards-meaningful-names-in-javascript) to get more information.